### PR TITLE
feat: nav bar added(#113)

### PR DESCRIPTION
**Pull request by Soham041201**
Added a basic Nav-bar with a basic links and dummy links

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/143"></a>

### CODE REVIEW

File: src/Components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function NavBar() {
+  return (
+    <nav className="navbar">
+      <h1>Blog website</h1>
```
Comment (nhcarrigan) [inline src/Components/NavBar.js:6]:
```suggestion
      <h1>LinkFree</h1>
```

File: src/index.css [hunk 2]
```diff
@@ -11,3 +11,35 @@ code {
...
+  text-decoration: none;
+  padding: 6px;
+  color:#333;
+}
+.navbar a:hover{
+  color:skyblue;
+  
+}
+
+/* =========Navbar Start==========*/
```
Comment (nhcarrigan) [inline src/index.css:45]:
```suggestion
/* =========Navbar End==========*/
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Comment by nhcarrigan** [comment]
Also please remove your changes to the `package.json` and `package-lock.json`

**Comment by Soham041201** [comment]
Made the required changes

**Review by Soham041201 — COMMENTED** [review]
Made Changes

**Review by qwe123coder — CHANGES_REQUESTED** [review]
hey, nice work :rocket: Thanks for your contribution
*  the files are Conflicted :( 
*  I suggest read [Resolving a merge conflictt](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/about-merge-conflicts)

**Comment by Soham041201** [comment]
I have resolved the conflict. There was just a small conflict of lines of code. Some one had written code on the same line on which I had written and also this code was not related to mine. Please review once

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]
Please try to add CSS from [Prime React ](https://www.primefaces.org/primereact/) as we are using it for whole project that will easier. 
If CSS is custom them keep it same else try to find some prime react alternative which will reduce css code

**Comment by Soham041201** [comment]
I did try to find a prime react component to add the Navbar. But I was not able to find one. That's why I did it through CSS.

**Comment by rohan-kulkarni-25** [comment]
> Okay Cool! Will make the changes

If not found such classes through prime react then its ok we can take it as plain css

**Review by krishdevdb — CHANGES_REQUESTED** [review]
This [component](https://www.primefaces.org/primereact/showcase/#/menubar) shall be used rather than custom css

**Review by krishdevdb — COMMENTED** [review]
Its okay for this pr, but in the future consider making different pull requests for different changes 👍🏻

**Comment by Soham041201** [comment]
> Its okay for this pr, but in the future consider making different pull requests for different changes 👍🏻
Yess! Just saw this component trying to inlude it. But unabe to apply styling. Should I just add the Nav-bar here and the styling can be done in other PR?

**Review by krishdevdb — APPROVED** [review]

**Comment by krishdevdb** [comment]
Seems good to me, let's wait for @nhcarrigan to review once?

**Comment by Soham041201** [comment]
> Seems good to me, let's wait for @nhcarrigan to review once?

Yess !

**Comment by nhcarrigan** [comment]
This will require a rebase :)

**Comment by Soham041201** [comment]
> This will require a rebase :)

Performed rebase :)

**Comment by krishdevdb** [comment]
What happened? So many commits?

**Comment by CodingSpecies** [comment]
I am not sure but I would close this pr and create a fresh one with one single commit will all the changes thanks

**Comment by Soham041201** [comment]
> I am not sure but I would close this pr and create a fresh one with one single commit will all the changes thanks

Yess Will do the same...

**Pull request closed without merging**